### ISSUE TITLE
UnusedUsesSniff: false positive in double-quoted string

### ISSUE THREAD

**Issue by schlndh** [slevomat/coding-standard#1441]
UnusedUsesSniff incorrectly reports an unused use for names that are used in double quoted string like this:

```php
<?php

namespace Foo\Bar;

use DateTimeImmutable;

$a = 'implode';
echo "aasdasd{$a(
    [DateTimeImmutable::class, 'format']
)}";
```

Which reports:

```
 5 | ERROR | [x] Type DateTimeImmutable is not used in this file.
```

However, if I put `$a(` and `[...` on the same line then it works as expected (i.e. no issue reported). The working example looks like this:

```php
<?php

namespace Foo\Bar;

use DateTimeImmutable;

$a = 'implode';
echo "aasdasd{$a( [DateTimeImmutable::class, 'format']
)}";
```
